ambari-server/src/main/java/org/apache/ambari/server/upgrade/UpgradeCatalog270.java

```diff
@@ -439,12 +439,12 @@ private void migrateRepoData() throws SQLException {
                 if (rootObject != null) {
                   JsonPrimitive osType = rootObject.getAsJsonPrimitive("OperatingSystems/os_type");
                   JsonPrimitive ambariManaged = rootObject.getAsJsonPrimitive("OperatingSystems/ambari_managed_repositories");
+                  String isAmbariManaged = (ambariManaged != null && ambariManaged.getAsBoolean()) ? "1" : "0";
```

Comment:
I'm wondering if it should default to 1 if `ambariManaged` is missing?  Not sure what the implications are in each case, but this would be consistent with the DB schema default:

https://github.com/apache/ambari/blob/14c55d316969e4777730b9bfcfa8f9b8e17032d7/ambari-server/src/main/resources/Ambari-DDL-Postgres-CREATE.sql#L180

Reply:
it makes sense; I modified the code accordingly (will submit my changes in a minute)

Reply:
Submitted (I needed to force push; sorry for this)